src/uu/more/src/more.rs

```diff
@@ -41,7 +41,7 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
 
     let matches = App::new(executable!())
         .version(VERSION)
-        .usage(&usage[..])
```

Comment:
Huh. I must have changed that subconsciously while reading it. Its certainly my personal preference (because in some languages that would copy the string) but I am not sure which is more idiomatic in rust.